tests/integration/olp-cpp-sdk-authentication/HereAccountOauth2Test.cpp

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <olp/core/porting/make_unique.h>
```

Comment:
only separate gtest/gmock includes and the rest you can put together.
No need to separate in groups.

Reply:
You mean to put all the OLP SDK includes in one group?

Reply:
Yes.